packages/labs/react/src/create-component.ts

```diff
@@ -241,6 +243,22 @@ export function createComponent<
     tag = tagName;
   }
 
+  // Warn users when web components use reserved React properties
+  if (DEV_MODE) {
+    for (const p in reservedReactProperties) {
```

Comment:
Should do a `for ... of ` loop through a set.

```suggestion
    for (const p of reservedReactProperties) {
```

Reply:
oh haha yah keys vs values doesn't matter too much in a set huh? will change :)

Reply:
done!